models/comment-model.js

```diff
@@ -0,0 +1,16 @@
+const connection = require("../db/connection");
+
+exports.selectCommentsByReview = async (review_id) => {
+  const firstQuery = await connection.query(
+    `SELECT * FROM comments WHERE comments.review_id = $1;`,
+    [review_id]
+  );
+
+  if (firstQuery.rowCount === 0) {
```

Comment:
with the correction on the valid review with no comments on it this if statement will no longer be valid